### PR TITLE
fix(progressbar): remove ngClass to avoid conflict with ngAnimate

### LINKED ISSUE #2670: Progressbar type property updates conflicts with ngAnimate causing no animation
When ngAnimate is imported updating a progress bar value and type at the same time results in the progress bar not animating.  Here is an example

http://plnkr.co/edit/uZr9XqsUgyT4sjDTKhLS?p=preview

It seems the best way to fix this is by this by removing ngCass from the bar templates and just update the bar element class directly in the directive.  I will create the pull request if this seems like a reasonable approach.

### PR DESCRIPTION
**Pull request by rgaskill**
fix #2670

### CODE REVIEW

**Comment by karianna** [comment]
Hi @rgaskill - Could you please add an accompanying test?

**Comment by rgaskill** [comment]
@karianna will do!

File: src/progressbar/progressbar.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ angular.module('ui.bootstrap.progressbar', [])
   max: 100
 })
 
-.controller('ProgressController', ['$scope', '$attrs', 'progressConfig', function($scope, $attrs, progressConfig) {
+.controller('ProgressController', ['$scope', '$element', '$attrs', '$animate', 'progressConfig', function($scope, $element, $attrs, $animate, progressConfig) {
+    $animate.enabled(false, $element);
```
Comment (rgaskill) [inline src/progressbar/progressbar.js]: This is the fix.

File: src/progressbar/test/progressbar.spec.js [hunk 2]
```diff
@@ -140,6 +167,15 @@ describe('progressbar directive', function () {
       expect(barEl).not.toHaveClass(BAR_CLASS + '-success');
       expect(barEl).toHaveClass(BAR_CLASS + '-warning');
     });
+
+    it('should not animate using ngAnimate', function() {
```
Comment (rgaskill) [inline src/progressbar/test/progressbar.spec.js]: new test for the progress bar

**Comment by rgaskill** [comment]
@karianna take a look.  The test turned out to be more interesting than I expected.  Let me know if there is a better way to test that the animation is turned off.

**Comment by karianna** [comment]
@chrisirhc - Can you comment on the choice of purgatory for this PR?

**Comment by chrisirhc** [comment]
Closing this as I have commented in #2670 that this is no longer valid for our next release.

**Pull request closed without merging**